Sources/ScipioKit/Producer/PIF/FrameworkComponentsCollector.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+import TSCBasic
+import PackageModel
+
+/// FileLists to assemble a framework bundle
+struct FrameworkComponents {
+    var name: String
```

Comment:
```suggestion
    var frameworkName: String
```

I'm not sure whether the name of "Component" or "Framework"